src/python/pants/backend/python/lint/pylint/subsystem.py

```diff
@@ -317,7 +317,13 @@ class PylintExportSentinel(ExportPythonToolSentinel):
     pass
 
 
-@rule
+@rule(
+    desc=(
+        "Determine all Python interpreter versions used by Pylint in your project (for "
+        "`export` goal)"
```

Comment:
The parenthetical doesn't add much I don't think... someone is already running `export` explicitly.

Reply:
I'm not totally sure I agree. They might have done `./pants generate-lockfiles export`. It was helpful w/ user's logs today to have the paranthetical